bin/load_db_scxa_analytics.sh

```diff
@@ -47,7 +45,7 @@ sed "s/<EXP-ACCESSION>/$lc_exp_acc/" $postgres_scripts_dir/02-create_partition_t
 psql $dbConnection
 # Create file with data
 matrixMarket2csv.R -m $matrix_path -r $genes_path -c $runs_path \
-                   -s 10000 -o $EXPERIMENT_MATRICES_PATH/expression2load.csv \
+                   -s 100000 -o $EXPERIMENT_MATRICES_PATH/expression2load.csv \
```

Comment:
Great that you have benchmarked it, lets go for that!

Reply:
Where did you benchmark it though? EBI filesystems tend to be awfully slow, so that might affect results. But happy to go with the new value given the benchmarks! I wonder if it might require adjusting the memory requirement for the cluster jobs... each record is tiny though.